lib/Config.cpp

```diff
@@ -26,24 +27,30 @@ void LibConfig::Config::initSocket()
   std::string socketPath = (std::filesystem::temp_directory_path() / "raven-os_service_libconfig.sock").string();
 
   socket->on<uvw::ErrorEvent>([](const uvw::ErrorEvent&e, uvw::PipeHandle&) {
-				std::cout << "Error" << std::endl;
-				throw std::exception();
-			      });
+    std::cout << "Error" << std::endl;
+    throw std::exception();
+  });
   socket->once<uvw::ConnectEvent>([](const uvw::ConnectEvent&, uvw::PipeHandle&) {
-				    std::cout << "All succeed" << std::endl;
-				 });
-  socket->on<uvw::DataEvent>([this](const uvw::DataEvent &data, uvw::PipeHandle &sock){
-			       std::cout << "Data received" << std::endl;
-			       try {
-				 parseResponse(json::parse(std::string(data.data.get(), data.length)));
-			       } catch (std::exception) {
-				 std::cout << "Error in data received" << std::endl;
-				 throw std::exception();
-			       }
-			     });
+    std::cout << "All succeed" << std::endl;
+  });
+  socket->on<uvw::DataEvent>([this](const uvw::DataEvent &data, uvw::PipeHandle &){
```

Comment:
Maybe call the argument `dataEvent` to avoid stuff like `data.data` (I'm aware I tend to not respect this rule)